### DISCUSSION TITLE
Struggling with my new FTXF35E and could use some help

### DISCUSSION THREAD

**Discussion by AchimWilden** [revk/xESP32-Faikout#63] [Q&A] [answered]
I have to admit that I didn't really think too much about choosing the model before I bought it...
The air conditioning is built in, the original Onecta module works and unfortunately doesn't inspire at all...
After a little googling I came across this really cool project and bought a Faikin module from Amazon.
I was worried it wouldn't be plug n play, but that's okay :) And that's exactly how it happened.
I looked around here and saw a few pictures and a few adapters and think I need something like that too.

Now I'm not an electronics developer but a software developer but unfortunately not in the embedded area either and so I'm having a hard time finding the right connector.

I have read that the plug on the Faikinmodul JST EHR-5 has a 2,5mm pitch.
Is [this ](https://www.ebay.de/itm/183466543983?)the right one? 

On the Daikin board, the WLAN module is on my S430 but it is the same small connector. I'm not sure what the good one is called.
Can someone give me a tip?
![image](https://github.com/revk/ESP32-Faikin/assets/73713500/3a0a1b4d-83ec-445c-86fe-f843f01f1d72)
![image](https://github.com/revk/ESP32-Faikin/assets/73713500/c181072b-151f-40ea-a403-ba5031293af7)

Thank you in advance for your help.
Achim

**Comment by revk** [comment]
Did you manage to sort this? The connectors they use are all JST.

**Comment by AchimWilden** [comment]
@revk, I didn't really get much further.
I didn't find the S430 connector, but it doesn't matter. Then I build a Y-piece from the original cable and just plug in one circuit board. In the best case I never have to go there again...

I've read a little more here under the discussions and now I'm not so sure what I'm doing anymore :-/
Okay, the S430 on which the Onecta WLAN module is connected also has 5 pins, but are they compatible? Can anyone confirm that?
And if so, how is the wiring between the S430 header and the faikin? Anyone have experience there? 

Cheers, Achim

**Reply by revk** [reply]
It is worth checking voltages. If they are compatible then maybe try it. If we are lucky it will be the same.

**Reply by AchimWilden** [reply]
@revk, of course, there is no harm in measuring the supply. 👍 

Is the pin assignment of the "standard Orecta wifi thing" known and can it be read somewhere? 
I won't be the first to want to replace the part with the Faikin...
Where can I find the pinout of the Faikin module and the tolerable voltage range?

**Reply by revk** [reply]
The Faikin is designed for S21 which is quite well known.

Notably (and annoyingly) pin 5 is GND.

Pin 1 (not always used) is 5V.

Pin 4 is typically 12 to 14V to power things

Pin 2 and 3 usually measure 5V, one is Tx and one is Rx with a 5V pull up.

So if you measure against pin 5 as GND and see pins 1-4 as 5V, 5V, 5V, 12V(ish), then a good chance it is the same as S21

**Reply by AchimWilden** [reply]
@revk thanks.
Then I'll go out with the multimeter and see if that's the case :)
I'll be in touch

**Comment by AchimWilden** [comment]
@revk, I think the S430 is not compatible.
 
![image](https://github.com/revk/ESP32-Faikin/assets/73713500/be13dc7d-2da6-4638-a347-9f1f2cba0567)
![image](https://github.com/revk/ESP32-Faikin/assets/73713500/de9ba7c4-7c45-4d73-b363-c9b17a9ae9a3)
Do you agree or do you still see opportunities?
Does it make sense to look at individual signals with the scope or are the levels already so off that it can't fit?

**Reply by revk** [reply, marked as answer]
OK maybe not compatible

**Comment by AchimWilden** [comment]
What a pity then I'll have to deal with it whether I can get to the onecta cloud in another way. *gagging*

Thank you for your support. The next AC is planned for next year, so I'll make sure in advance that an S21 base is available. I already have the Faikin module :)
Still a very cool project. I looked around in the documentation and in particular in the source code. I can see that a lot of time and evolution have been invested. *respect*
